openedx/core/djangoapps/credit/models.py

```diff
@@ -111,6 +111,26 @@ class CreditProvider(TimeStampedModel):
         )
     )
 
+    eligibility_email_message = models.TextField(
+        null=True,
```

Comment:
Do we need to all both NULL and empty string values here?  Why not just the empty string?